tests/capi/unittest_capi_service.cc

```diff
@@ -12,143 +12,102 @@
 #include <ml-api-service.h>
 
 /**
- * @brief Set the pipeline description with given name
+ * @brief Test base class for Database of ML Service API.
  */
-TEST (nnstreamer_capi_service, set_pipeline_description_0_p)
+class MLServiceDBTest:
+    public::testing::Test
 {
-  int ret = 0;
-  gchar *pipeline, *test_model;
-  const gchar *key = "ServiceName";
-  const gchar *root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
-
-  /* supposed to run test in build directory */
-  if (root_path == NULL)
-    root_path = "..";
-
-  test_model =
-      g_build_filename (root_path, "tests", "test_models", "models",
-      "add.tflite", NULL);
-  EXPECT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS));
+protected:
+  const gchar * root_path;
```

Comment:
Plz check coding style in this PR (unnecessary space between '*' and 'variable name'):
gchar * root_path > gchar *root_path

Reply:
After talking with @jaeyun-jung, I realized that gst-indent script is not suitable for C++ format.
I will submit the next version ASAP.